mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/norg2/Norg2ClientImpl.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.mulighetsrommet.api.clients.norg2
+
+import io.ktor.client.call.*
+import io.ktor.client.engine.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.request.*
+import no.nav.mulighetsrommet.api.domain.Norg2Enhet
+import no.nav.mulighetsrommet.api.setup.http.httpJsonClient
+import org.slf4j.LoggerFactory
+
+class Norg2ClientImpl(
+    private val baseUrl: String,
+    clientEngine: HttpClientEngine = CIO.create()
+) : Norg2Client {
+    private val log = LoggerFactory.getLogger(javaClass)
+    private val client = httpJsonClient(clientEngine)
+    override suspend fun hentEnheter(): List<Norg2Enhet> {
+        return try {
+            val response = client.get("$baseUrl/enhet")
+            response.body()
+        } catch (exe: Exception) {
+            log.error("Klarte ikke hente enheter fra NORG2. Konsekvensen er at oppdatering av enheter mot database ikke blir kjørt")
+            emptyList()
```

Comment:
Tror det er bedre å la en eventuell exception boble videre opp. Da tror jeg vil dbscheduler markere jobben som feilet, i stedet for at det blir en success :thinking:

Reply:
Godt poeng!